### ISSUE TITLE
support kcptun more easily

### ISSUE THREAD

**Issue by zhiyongLiu1114** [shadowsocks/shadowsocks-windows#2933]
can you have a look at https://github.com/yangfeicheung/Shadowsocks-X, and develope the ss app more easily to set kcptun? both for macos and windows, it hard to follow https://github.com/shadowsocks/ShadowsocksX-NG/wiki/SIP003-Plugin to finish my kcptun set for me.
 thanks very much anyway

**Comment by studentmain** [comment]
"support kcptun more easily" will eventually became "bundle every plugin", that's not our goal. If you have idea about plugin API, open an issue in shadowsock-org instead.
